<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
background-color: #FFCC01;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="headline">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Radio Space Bold</family>
          <pointsize>30</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
background-color: #FFCC01;
border-bottom: 2px solid black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QGridLayout" name="ListLeft">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_RankLeft">
            <property name="font">
             <font>
              <family>Radio Space Bold</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 0, 6);
border-top: none;</string>
            </property>
            <property name="text">
             <string>Pořadí</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_RoundLeft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Radio Space Bold</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 0, 6);
border-top: none;</string>
            </property>
            <property name="text">
             <string>Kol</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_NameLeft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Radio Space Bold</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 0, 6);
border-top: none;</string>
            </property>
            <property name="text">
             <string>Jméno</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="middleLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="ListRight">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_RankRight">
            <property name="font">
             <font>
              <family>Radio Space Bold</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 0, 6);
border-top: none;</string>
            </property>
            <property name="text">
             <string>Pořadí</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_NameRight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Radio Space Bold</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 0, 6);
border-top: none;</string>
            </property>
            <property name="text">
             <string>Jméno</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_RoundRight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Radio Space Bold</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 0, 6);
border-top: none;</string>
            </property>
            <property name="text">
             <string>Kol</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="action_changeSettings"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Print"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="action_TeamEdit"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Testovací</string>
    </property>
    <addaction name="actionBestByRoundsDesc"/>
    <addaction name="actionBestByRoundsAsc"/>
    <addaction name="actionPrintByRoundsDesc"/>
    <addaction name="actionPrintByRoundsAsc"/>
    <addaction name="action_PrintByRounds"/>
    <addaction name="actionBestByTimeDesc"/>
    <addaction name="actionBestByTimeAsc"/>
    <addaction name="actionPrintByTimeDesc"/>
    <addaction name="actionPrintByTimeAsc"/>
    <addaction name="action_PrintByTeam"/>
    <addaction name="actionXmlExport"/>
    <addaction name="actionXmlImport"/>
   </widget>
   <widget class="QMenu" name="menu_azen">
    <property name="title">
     <string>Řazení</string>
    </property>
    <addaction name="action_SortByTimeAsc"/>
    <addaction name="action_SortByTimeDesc"/>
    <addaction name="action_SortByRoundsAsc"/>
    <addaction name="action_SortByRoundsDesc"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuData"/>
   <addaction name="menu_azen"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="myStatusBar"/>
  <action name="action_Start">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionBestByRoundsDesc">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>bestByRoundsDesc</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBestByRoundsAsc">
   <property name="text">
    <string>bestByRoundsAsc</string>
   </property>
  </action>
  <action name="actionPrintByRoundsDesc">
   <property name="text">
    <string>printByRoundsDesc</string>
   </property>
  </action>
  <action name="actionPrintByRoundsAsc">
   <property name="text">
    <string>printByRoundsAsc</string>
   </property>
  </action>
  <action name="actionBestByTimeDesc">
   <property name="text">
    <string>bestByTimeDesc</string>
   </property>
  </action>
  <action name="actionBestByTimeAsc">
   <property name="text">
    <string>bestByTimeAsc</string>
   </property>
  </action>
  <action name="actionPrintByTimeDesc">
   <property name="text">
    <string>printByTimeDesc</string>
   </property>
  </action>
  <action name="actionPrintByTimeAsc">
   <property name="text">
    <string>printByTimeAsc</string>
   </property>
  </action>
  <action name="actionXmlExport">
   <property name="text">
    <string>xmlExport</string>
   </property>
  </action>
  <action name="actionXmlImport">
   <property name="text">
    <string>xmlImport</string>
   </property>
  </action>
  <action name="action_PrintByTeam">
   <property name="text">
    <string>printByTeam</string>
   </property>
  </action>
  <action name="action_SortByTimeDesc">
   <property name="text">
    <string>seřadit podle nejhoršího času</string>
   </property>
  </action>
  <action name="action_SortByTimeAsc">
   <property name="text">
    <string>seřadit podle nejlepšího času</string>
   </property>
  </action>
  <action name="action_SortByRoundsDesc">
   <property name="text">
    <string>seřadit podle počtu kol sestupně</string>
   </property>
  </action>
  <action name="action_SortByRoundsAsc">
   <property name="text">
    <string>seřadit podle počtu kol vzestupně</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Uložit stav</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>Načíst stav</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>Tisk</string>
   </property>
  </action>
  <action name="action_TeamEdit">
   <property name="text">
    <string>Editovat týmy</string>
   </property>
  </action>
  <action name="action_changeSettings">
   <property name="text">
    <string>Změnit nastavení</string>
   </property>
  </action>
  <action name="action_PrintByRounds">
   <property name="text">
    <string>printByRounds</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
